src/Core/Checker.cpp

```diff
@@ -182,7 +179,8 @@ void Checker::onCompilationKilled()
     disconnect(compiler, &Compiler::compilationErrorOccurred, this, &Checker::onCompilationErrorOccurred);
 }
 
-void Checker::onRunFinished(int index, const QString &, const QString &err, int exitCode, int, bool tle)
+void Checker::onRunFinished(int index, const QString & /*unused*/, const QString &err, int exitCode, int /*unused*/,
```

Comment:
Every argument in a function definition should be named or marked /*unused*/ to signify this argument isn't used.